packages/suite/src/components/suite/modals/TransactionDetail/index.tsx

```diff
@@ -55,10 +55,12 @@ type Props = {
 
 const TransactionDetail = (props: Props) => {
     const { tx } = props;
+
     const { blockchain, transactions } = useSelector(state => ({
         blockchain: state.wallet.blockchain[tx.symbol],
         transactions: state.wallet.transactions.transactions,
     }));
+
```

Comment:
this is where you should do:
`blockchain.url ? `${blockchain.url}/tx/${tx.txid}` : getBlockExplorerUrl(tx)`

Reply:
I can do it for `BasicDetail` component, but `ChainedTxs` receives multiple txs in array and then mapping it in render

Reply:
2ea85cd